pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>net.i2p.crypto</groupId>
   <artifactId>eddsa</artifactId>
-  <version>0.2.0</version>
+  <version>0.3.0-SNAPSHOT</version>
```

Comment:
The next release will be `0.2.1`, which I will do shortly. I'll also start pushing snapshot version changes in future :smile: